Cartfile

```diff
@@ -5,3 +5,4 @@ github "kif-framework/KIF"
 github "st3fan/SWXMLHash"
 github "rs/SDWebImage"
 github "norio-nomura/Base32"
+github "DaveWoodCom/XCGLogger" ~> 1.8
```

Comment:
This locks us into 1.8, meaning `carthage update` won't bump the version unless we do it manually, right? None of our other dependencies do this -- any reason we want to do that for `XCGLogger` specifically?